src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -4750,17 +4750,42 @@ public static void enableAutoCompaction() {
         }
     }
 
-    public void enableNode() {
+    public void internalDisableNode() {
```

Comment:
can this be made protected?

Reply:
Sadly the `Directories` class is in a separate package :(

Reply:
I wil rename to `unsafeDisableNode`